src/goe/config/option_descriptions.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python3
+
+# Copyright 2016 The GOE Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" option_descriptions: Library of constants defining descriptions for options.
+    In the future we expect to refactor all option processing, including descriptions, and this module will
+    may become redundant at that time.
+"""
+
+DATA_SAMPLE_PARALLELISM = (
+    "Degree of parallelism to use when sampling RDBMS data for columns with no precision/scale properties. "
+    "Values of 0 or 1 will execute the query without parallelism"
+)
+
+RESET_BACKEND_TABLE = "Remove backend data table. Use with caution - this will delete previously offloaded data for this table!"
+
+REUSE_BACKEND_TABLE = (
+    "Allow Offload to re-use an empty backend table when there is already Offload metadata. "
+    "This may be useful if a backend table had data removed by an administrator and a re-offlaod is required"
```

Comment:
Typo: `re-offlaod`.

Also, there's an inconsistency between the use of full-stops for option descriptions. None of the options here have a full-stop (other than RESET with it's exclamation mark). But when doing an `offload -h` the inconsistency becomes clear (note the purge and reuse options and those around them):

```
  -f, --force           Replace tables/views as required. Use with caution.
  --create-backend-db   Automatically create backend databases. Either use
                        this option, or ensure databases matching 1) the
                        Oracle schema and 2) the Oracle schema with suffix
                        _load already exist.
  --reset-backend-table
                        Remove backend data table. Use with caution - this
                        will delete previously offloaded data for this table!
  --reset-hybrid-view   Reset Incremental Partition Append or Predicate-Based
                        Offload predicates in the hybrid view.
  --reuse-backend-table
                        Allow Offload to re-use an empty backend table when
                        there is already Offload metadata. This may be useful
                        if a backend table had data removed by an
                        administrator and a re-offlaod is required
  --purge               Include PURGE operation (if appropriate for the
                        backend system) when removing data table with --reset-
                        backend-table. Use with caution
  --equal-to-values=EQUAL_TO_VALUES
                        Offload partitions with a matching partition key list.
                        This option can be included multiple times to match
                        multiple partitions.
```

Reply:
Fixed the typo, thanks.
The inconsistency with full-stops is throughout all of the options. My preference is to have no trailing full-stop but include mid text when multiple sentences. However I'm not going to tackle this now, I've created an issue instead: https://github.com/gluent/goe/issues/173
Feel free to add comments to the issue if there are other inconsistencies of a similar nature.